content/OTP/Guides/OTP_Walk-through.adoc

```diff
@@ -181,13 +187,9 @@ b) Verify submitted OTPs. Add to your client:
 
 The `otp=` value is the OTP from the YubiKey that the user inserts.
 
-c) Verify user login. Add to your client:
-
-	assert otp[:12] == user.yubikey_id
-
-`12` indicates the first 12 numbers from the `otp=` field. On the YubiCloud  validation server, this value is compared with the YubiKey ID that is associated with the user.
```

Comment:
Chris Halos: not a fan of the usage of "YubiKey ID" when they're talking about the "YubiKey Public ID" but it's used quite a few times in here, and I guess if the code essentially calls it yubikey_ID, maybe this is alright? :man-shrugging: